client/src/app/site/interaction/components/call/call.component.ts

```diff
@@ -47,10 +48,7 @@ export class CallComponent extends BaseComponent implements OnInit, AfterViewIni
     }
 
     private dominantSpeaker: string;
-    private members = {};
-    public get memberList(): string[] {
-        return Object.keys(this.members);
-    }
+    public members: ConferenceMemberCollection = {};
```

Comment:
I like, that there are type-information are added. But if the `members`-variable is not changed from the outside, it should be kept private. Please make it accessible from the outside by a `get`-function (if it won't be changed from there).

Reply:
seems it was not necessary at all since members can be accessed using async pipe